dwitter/static/js/scrolling.js

```diff
@@ -1,13 +1,12 @@
 window.onload = function() {
-  var infinite = new Waypoint.Infinite({
+  // eslint-disable-next-line no-new
+  new Waypoint.Infinite({
```

Comment:
The Airbnb ESLint config has a rule against calling `new` just for its side effects. In this case, it seems easier to ignore the rule than to change the Waypoints API.